### ISSUE TITLE
Users can update the Desktop app after 1st install

### ISSUE THREAD

**Issue by 0xSage** [janhq/jan#188]
@louis-jan @dan-jan do we need to implement a "update available / download latest version" feature in order to let users who download the Desktop app now be able to keep up with changes? A bit of a chicken and egg problem if this isn't included for the 1st batch of public users

**Comment by dan-jan** [comment]
Yes, I think Electron has an embedded updater. This is a definite must-have
before first release
